### ISSUE TITLE
Current/Active Breadcrumb should have `aria-current="page"`

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#659]
Like Modus Bootstrap has - see first example:
https://modus-bootstrap.trimble.com/components/breadcrumbs/
